proposal.rst

```diff
@@ -259,7 +313,8 @@ For the avoidance of doubt this includes all contributions to our wiki, mailing
 lists and documentation, including anything you write in your project's wiki
 pages.
 
-FILLME
+Yes, I agree to dual-license all your contributions to Xapian under the GNU
```

Comment:
You probably mean "my contributions" rather than "your contributions" :-)

Reply:
Oops! Thank you.